### PR TITLE
Run unit tests in CI

### PR DESCRIPTION
**Pull request by pschmitt**
- Unit tests via GH Actions
- Remove travic CI config

### CODE REVIEW

File: .github/workflows/zunit.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo apt install -y zsh
+
+          mkdir bin
+          curl -L https://raw.githubusercontent.com/molovo/revolver/master/revolver > bin/revolver
+          curl -L https://raw.githubusercontent.com/molovo/color/master/color.zsh > bin/color
```
Comment (alichtman) [inline .github/workflows/zunit.yaml:17]: Not a huge fan of curling dependencies that are out of our control (account might be deleted and replaced with something malicious). This is running on CI but we're not doing this with regular zinit, right?

Comment (pschmitt) [reply .github/workflows/zunit.yaml:17]: > Not a huge fan of curling dependencies that are out of our control (account might be deleted and replaced with something malicious)

Agreed. I just translated the original Travis CI config to a GH workflow. I can pin commits and/or releases though. Makes sense to do so, yup. Will do.

> This is running on CI but we're not doing this with regular zinit, right?
No zinit here, no. That's another repo.

Comment (alichtman) [reply .github/workflows/zunit.yaml:17]: I meant that we should avoid curling "untrusted" deps in zinit.

Comment (pschmitt) [reply .github/workflows/zunit.yaml:17]: Oh. You merged it...

Comment (pschmitt) [reply .github/workflows/zunit.yaml:17]: :eyes: #10

**Pull request merged**